pkg/v1/tkg/test/tkgctl/tkgs/tkgs_workload_cluster_test.go

```diff
@@ -171,6 +245,17 @@ func createClusterClassBasedClusterTest(tkgctlClient tkgctl.TKGClient, deleteClu
 	}
 }
 
+// deleteClusterClassBasedCluster deletes workload cluster
```

Comment:
comment is slightly off. This deletes a class based workload cluster. If you can don't include comment. I don't think go doc will pick it up anyways. (lower case function). Is pretty obvious from the function name what it does.